scripts/get-screen-shots.pl

```diff
@@ -186,11 +216,12 @@ ($driver)
 # Example:
 #    'login.png' => { h => 520, w => 520,  pre => \&preprocess_dummy, post => \&postprocess_dummy},
 my %processing_config = (
-    'example'     => { h => 520, w => 520,  pre => \&preprocess_dummy, post => \&postprocess_dummy},  # Example should not be used.
-    'default'     => { h => 820, w => 1200},  # The default screen resize values if there is no match file name match
-    'login.png'   => { h => 520, w => 520},   # No need for a lot of empty space
-    'welcome.png' => { h => 700, w => 1280, pre => \&login},  # Make the screen wider since it includes the welcome text
-    'preferences-preferences.png' => {h => 820, w => 1200, pre => \&select_preferences_tab},
+    'default'                     => { h => 820,  w => 1200},  # The default screen resize values if there is no match file name match
+    $login_screenshot_file_name   => { h => 520,  w => 520},   # No need for a lot of empty space
+    $welcome_screenshot_file_name => { h => 700,  w => 1280, pre => \&login},  # Make the screen wider since it includes the welcome text
+    'preferences-preferences.png' => { h => 820,  w => 1200, pre => \&select_preferences_tab},
+    'system--defaults.png'        => { h => 820, w => 1300, post => \&post_process_system_defaults},
```

Comment:
The newly added defaults screen has dimensions `2118px | H: 1450px` (see above). How do you explain these dimensions vs the image dimensions given in this line?

Reply:
> That dimension comes from Selenium.

Yes, I can imagine it does. What's the difference between the two - if there is any?